components/badge/badge.css

```diff
@@ -0,0 +1,116 @@
+@import url(/stylesheets/default.css);
+
+main header {
+  height: 100%;
+}
+.badge-link {
+  background-color: var(--extra-color);
+}
+.badge-1 {
+  margin: var(--gap-large);
+}
+.badges-icon {
+  display: flex;
+  justify-content: center;
+  padding: var(--gap-small);
+}
+.badges-icon a {
+  background-color: white;
+  color: black;
+}
+.badges-icon p {
+  font-size: 0.8rem;
+  text-align: center;
+  color: white;
+  background-color: var(--primary-color);
+}
+.icon-bell {
+  position: relative;
+}
+.icon-bell p {
+  position: absolute;
+  top: -0.9rem;
```

Comment:
Not good practise to use rems below 1, convert to absolute units

Reply:
yes thank you i've fixed ti can you review it again